packages/cojson/src/coValues/coMap.ts

```diff
@@ -374,6 +365,10 @@ export class RawCoMap<
     value: Shape[K],
     privacy: "private" | "trusting" = "private",
   ): void {
+    if (this.isTimeTravelEntity()) {
+      throw new Error("Cannot set value on a time travel entity");
+    }
```

Comment:
Added these checks because it would be too weird/complex to add transactions on a time traveled entity and I think it's fair to consider it a "frozen" entity.